environment-model/environment-model_shacl.ttl

```diff
@@ -3,134 +3,286 @@
+            sh:name "boundingBox"@en ;
+            sh:node environment-model:BoundingBoxShape ;
+            sh:minCount 1 ;
+            sh:maxCount 1 ;
+            sh:order 5 ;
+            sh:path environment-model:boundingBox ],
+        [ skos:example "Autobahn A8, Großraum Ingolstadt" ;
+            sh:datatype xsd:string ;
+            sh:description "Description of the location, dependen if it is a more trajectory style data or area style data"@en ;
+            sh:message "Validation of relationOrArea failed!"@en ;
+            sh:maxCount 1 ;
+            sh:name "relationOrArea"@en ;
+            sh:order 4 ;
+            sh:path environment-model:relationOrArea ] ;
+    sh:targetClass environment-model:ProjectLocation .     
+
+environment-model:FormatShape a sh:NodeShape ;
+    sh:property [ skos:example "Unreal DataSmith, Autodesk FBX, OpenSceneGraph" ;
+            sh:datatype xsd:string ;
+            sh:description "Format type definition"@en ;
+            sh:in ("Autodesk FBX" "GLTF" "OpenSceneGraph" "Unreal DataSmith") ;
+            sh:maxCount 1 ;
+            sh:message "Validation of type failed!"@en ;
+            sh:name "type"@en ;
+            sh:order 0 ;
+            sh:path environment-model:type ],
+        [ skos:example "SDK 4.25" ;
+            sh:datatype xsd:string ;
+            sh:description "Version of data format"@en ;
+            sh:maxCount 1 ;
             sh:message "Validation of version failed!"@en ;
-            sh:minCount "1"^^xsd:unsignedInt ;
             sh:name "version"@en ;
-            sh:path environment-model:environment-model_data_version ],
+            sh:order 1 ;
+            sh:path environment-model:version ] ;
+    sh:targetClass environment-model:Format .
+
+environment-model:ProjectShape a sh:NodeShape ;
+    sh:property [ skos:example "7.7" ;
+            sh:datatype xsd:string ;
+            sh:description "Tool for the creation of the data"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of creationVersion failed!"@en ;
+            sh:name "creationVersion"@en ;
+            sh:order 4 ;
+            sh:path environment-model:creationVersion ],
+        [ skos:example "Trian3DBuilder" ;
+            sh:datatype xsd:string ;
+            sh:description "Tool for the creation of the data"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of creationSource failed!"@en ;
+            sh:name "creationSource"@en ;
+            sh:order 3 ;
+            sh:path environment-model:creationSource ],
+        [ skos:example "Unreal" ;
+            sh:datatype xsd:string ;
+            sh:description "Name of the visual system"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of softwareName failed!"@en ;
+            sh:name "softwareName"@en ;
+            sh:order 0 ;
+            sh:path environment-model:softwareName ],
         [ skos:example "Epic" ;
```

Comment:
It is called "Epic Games". We really need to be more precise with a lot of things here

Reply:
fixed